plugins/line-highlight/prism-line-highlight.js

```diff
@@ -195,6 +195,8 @@
 				});
 			}
 
+			line.style.width = pre.scrollWidth + 'px';
```

Comment:
This mutates the DOM, so it has to be pushed to `mutateActions`.

```suggestion
			mutateActions.push(function () {
				line.style.width = pre.scrollWidth + 'px';
			});
```